src/main/java/org/nuxeo/ecm/csv/CSVImporterWork.java

```diff
@@ -74,6 +74,7 @@
 import org.nuxeo.ecm.core.schema.DocumentType;
 import org.nuxeo.ecm.core.schema.SchemaManager;
 import org.nuxeo.ecm.core.schema.types.ComplexType;
+import org.nuxeo.ecm.core.schema.types.CompositeType;
```

Comment:
The more general CompositeType makes the computePropertiesMap() method more extensible. For example, it allows the method to be passed a dynamic facet in addition to a document type. This makes it much easier for custom CSVImporterWork extensions to support dynamic facets, for example.